UI/webpack.config.js

```diff
@@ -67,11 +67,11 @@ if (TARGET !== "readme") {
 
     /* eslint-disable-next-line no-inner-declarations */
     function globCssEntries(globPath) {
-        var files = glob.sync(globPath);
-        var entries = {};
```

Comment:
With this change, we can remove the "no-inner-declaratios" eslint directive, no?

Reply:
No, the function is not defined at root but in the  application section, because README has no use for that.